src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImplTest.java

```diff
@@ -2718,4 +2725,39 @@ void shouldRetrieveOrganisationsWithEmptyProfileIdList() {
         assertThat(result).isNotNull();
         assertThat(result.getOrganisationInfo()).isNullOrEmpty();
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    void test_deleteUserForOrganisation() throws JsonProcessingException {
+        final HttpStatus expectedHttpStatus = HttpStatus.OK;
+        String deleteBody = new ObjectMapper().writeValueAsString(new NewUserResponse());
+        when(professionalUserRepositoryMock.findByEmailAddress(anyString())).thenReturn(professionalUser);
+        doNothing().when(userAttributeRepositoryMock).deleteByProfessionalUserId(professionalUser.getId());
+        doNothing().when(professionalUserRepositoryMock).delete(professionalUser);
+
+        when(userProfileFeignClient.deleteUserProfile(any()))
+            .thenReturn(Response.builder().status(STATUS_CODE_204).reason("OK").body(deleteBody, UTF_8)
+            .request(mock(Request.class)).build());
+
+        assertThat(deleteOrganisationResponse).isNotNull();
+        assertThat(deleteOrganisationResponse.getStatusCode()).isEqualTo(STATUS_CODE_204);
+        verify(userProfileFeignClient, times(0)).deleteUserProfile(any());
+
+        DeleteUserResponse deleteUserResponse =
+            new DeleteUserResponse(204, "The organisation has deleted successfully");
```

Comment:
wrong message you are trying delete user : The organisation has deleted successfully